resources/js/components/Product/AddToCart.vue

```diff
@@ -244,8 +244,10 @@ export default {
                 selectedOptions.push(btoa('configurable/' + optionId + '/' + optionValue))
             })
 
-            Object.entries(this.customSelectedOptions).forEach(([optionId, optionValue]) => {
-                selectedOptions.push(btoa('custom-option/' + optionId + '/' + optionValue))
+            Object.entries(this.customSelectedOptions).forEach(([optionId, optionValues]) => {
+                ;[optionValues].flat().forEach((optionValue) => {
```

Comment:
Where is the `;` coming from?

Reply:
It seems to be a prettier check doing that.
[`8fabb2f` (#484)](https://github.com/rapidez/core/pull/484/commits/8fabb2f7ead56cbd86fb88f5201a75e0d1b6a0a4)

Possibly a safety thing where the browser could interpret the `{[` differently than expected